src/routes/participants-assignments.svelte

```diff
@@ -35,13 +36,12 @@
 		const participant = $participants[index];
 		const assignee = `${participant.firstName} ${participant.lastName}`;
 		const secret = btoa(`${assignment.firstName} ${assignment.lastName}`);
-		const params = new URLSearchParams({ assignee, secret }).toString();
-		navigator.clipboard.writeText(`${document.location.origin}/secret/?${params}`);
+		createLink(document.location.origin, {assignee, secret})
```

Comment:
Let's have `createLink` return a string and we will use the `navigator.clipboard.writeText` inside this `copyLink` function.